test/integration/framework/master_utils.go

```diff
@@ -521,3 +523,10 @@ func FindFreeLocalPort() (int, error) {
 	}
 	return port, nil
 }
+
+// SharedEtcd create an independent etcd server instance in-process.
+func SharedEtcd(t *testing.T) *storagebackend.Config {
```

Comment:
We don't use `t`. Can we drop it? Less mutable state to grok.

Reply:
fixed